scout/commands/export/case.py

```diff
@@ -21,6 +21,7 @@
 @click.option("-f", "--finished", is_flag=True, help="archived or solved")
 @click.option("--causatives", is_flag=True, help="Has causative variants")
 @click.option("--research-requested", is_flag=True, help="If research is requested")
+@click.option("--rerun-monitor", is_flag=True, help="If case rerun monitoring is requested")
```

Comment:
"If case rerun monitoring is requested", from just reading this text I do not really understand what it does, can it be more clear? Now it sounds like you ask if a specific case is monitored, as I understand it you will get all monitored cases returned together with some metadata.

Reply:
Yeah, I just followed the style we have for that endpoint, but I agree; it's not easy to parse without examples. I've added a few words; hopefully it's a little better.